hp_optim/run_learning.py

```diff
@@ -141,15 +148,75 @@ def read_reward_from_log(log_dir: pathlib.PurePath):
     return eprewmean
 
 
+# TODO maybe better to implement this in a class
+def run_learning(config_file, env, proc_backend):  # TODO type hints
```

Comment:
TODO: why not now ? :)

Reply:
Indeed, I missed that TODO.  Thanks for pointing it out!

Reply:
Added type hints and docstring.
I plan to address the other TODO in a separate PR (with some bigger refactoring of the script) to not delay merging this "multiple runs" feature.